ios/XMTPModule.swift

```diff
@@ -135,11 +136,21 @@ public class XMTPModule: Module {
 
         // Create a client using its serialized key bundle.
         AsyncFunction("createFromKeyBundle") { (keyBundle: String, environment: String) -> String in
-            let bundle = try PrivateKeyBundle(serializedData: Data(base64Encoded: Data(keyBundle.utf8))!)
-            let options = XMTP.ClientOptions(api: apiEnvironments[environment] ?? apiEnvironments["dev"]!)
-            let client = try await Client.from(bundle: bundle, options: options)
-            self.clients[client.address] = client
-            return client.address
+            do {
+                guard let keyBundleData = Data(base64Encoded: keyBundle),
+                    let bundle = try? PrivateKeyBundle(serializedData: keyBundleData) else {
+                    throw Error.invalidKeyBundle
+                }
```

Comment:
Seems like a change like this should probably also be done on the android side.

Reply:
Although Android is gracefully failing, I added a try-catch block to guarantee we keep exceptions handled.